test/config/ra.json

```diff
@@ -106,9 +106,7 @@
 				}
 			}
 		},
-		"features": {
-			"CheckRenewalExemptionAtWFE": true
```

Comment:
This should be left in place for now, to reflect the fact that we haven't actually deleted the feature flag from our production configs yet -- that can't happen until _after_ the rest of this change has been merged and deployed.